perfrunbook/utilities/find_and_list_jar_with_so.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [[ $(id -u) -ne 0 ]]; then
+  echo "Must run with sudo privileges"
+  exit 1
+fi
+
+i=$(find / -name '*.jar' -type f -print)
+i=($i)
+
+mkdir -p jar_dir
```

Comment:
not a dir in /tmp?

Reply:
Its expected to run from the utilities directory, but can move it to /tmp to be cleaner.